src/xclim/indices/_conversion.py

```diff
@@ -492,6 +493,48 @@ def saturation_vapor_pressure(
     return e_sat
 
 
+@declare_units(tas="[temperature]", hurs="[]", ice_thresh="[temperature]")
+def vapor_pressure_deficit(
+    tas: xr.DataArray,
+    hurs: xr.DataArray,
+    ice_thresh: Quantified | None = None,
+    method: str = "sonntag90",
+) -> xr.DataArray:
+    """
+    Vapour pressure deficit.
+
+    The measure of the moisture deficit of the air.
+
+    Parameters
+    ----------
+    tas : xarray.DataArray
+        Mean daily temperature.
+    hurs : xarray.DataArray
+        Relative humidity.
+    ice_thresh : Quantified, optional
+        Threshold temperature under which to switch to equations in reference to ice instead of water.
+        If None (default) everything is computed with reference to water.
+    method : {"goffgratch46", "sonntag90", "tetens30", "wmo08", "its90"}
+        Which method to use, see notes of :py:func:`saturation_vapor_pressure`.
+        Default is "sonntag90".
```

Comment:
"Method used to compute saturation_vapor_pressure" or something like that? I know the function is referenced, but maybe also put it in the sentence? I don't know, if it's similar to what we always do  in xclim, never mind that